prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    const fullName = 'Jay Park';
+    console.log('value of fullName:', fullName);
+    const numberOfPets = '0';
+    console.log('value of numberOfPets:', numberOfPets);
+    const likesVegetables = 'true';
+    console.log('value of likesVegetables:', likesVegetables);
+    const nothing = 'null';
```

Comment:
For each of `0`, `true`, and `null`, you want to leave out the quotes so that the JavaScript engine doesn't think you mean string values.